app/models/katello/concerns/host_managed_extensions.rb

```diff
@@ -51,6 +51,9 @@ def update_os_from_facts
         before_save :correct_puppet_environment
         before_validation :correct_kickstart_repository
 
+        after_validation :queue_reset_subscription_status
+        register_rebuild(:queue_reset_subscription_status, N_("Subscription_Status"))
```

Comment:
Some of the references to "subscription status" need to be reworded because we are doing more than that. Something like 'Content Host Status' ?